content/changelog/2024-01-22-matomo-5.0.1.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Matomo 5.0.1 is available
+date: 2024-01-22
+tags:
+  - addons
+  - matomo
+authors:
+  - name: David Legrand
+    link: https://github.com/davlgd
+    image: https://github.com/davlgd.png?size=40
+  - name: Rémi Collignon-Ducret
+    link: https://github.com/miton18
+    image: https://github.com/miton18.png?size=40
+description: You can now deploy Matomo 5.0.1 on Clever Cloud
+excludeSearch: true
+---
+
+Matomo 5.0 was released mid-december, version 5.0.1 is now available. Since then, we've worked on the support of this new branch on Clever Cloud. It's done! You can deploy this latest version right now from our [Console](https://console.clever-cloud.com) or [Clever Tools](https://github.com/CleverCloud/clever-tools). 
+
+Existing customers instances are already up-to-date.
```

Comment:
From customer point of view, they ordered an addon
)
```suggestion
Existing customers addons are already up-to-date.
```

Reply:
The chosen syntax is `add-on` in the docs